exersices/ex04/myrand.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'sebastian kihle'
+__email__ = 'sebaskih@nmbu.no'
+
+
+class LCGRand:
+    def __init__(self, seed):
+        self.seed = seed
+        self.counter = 0
+        self.r = [seed]
```

Comment:
No need to store all previous values in a list, you just need the previous value to calculate the new one